pkgs/by-name/az/az-pim-cli/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+  versionCheckHook,
+}:
+buildGoModule (finalAttrs: {
+  pname = "az-pim-cli";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "netr0m";
+    repo = "az-pim-cli";
+    rev = "v${finalAttrs.version}";
+    sha256 = "sha256-Zi/DNTroMews4PvPCeLWSq74xWdZ22NO2VtmW91zcfs=";
+  };
+
+  vendorHash = "sha256-g4NcRNmHXS3mOtE0nbV96vFFoVzGFbAvcj/vkdXshoU=";
+
```

```suggestion

  nativeBuildInputs = [
    installShellFiles
  ];

  postInstall = lib.optionalString (stdenv.hostPlatform.emulatorAvailable buildPackages) (
    let
      emulator = stdenv.hostPlatform.emulator buildPackages;
    in
    ''
      installShellCompletion --cmd az-pim-cli \
        --bash <(${emulator} $out/bin/az-pim-cli completion bash) \
        --fish <(${emulator} $out/bin/az-pim-cli completion fish) \
        --zsh <(${emulator} $out/bin/az-pim-cli completion zsh)
    ''
  );
```

This adds shell completions.